pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -705,14 +705,19 @@ func newKVEventToRowConsumer(
 	knobs TestingKnobs,
 	topicNamer *TopicNamer,
 ) kvEventConsumer {
-	rfCache := newRowFetcherCache(
+	rfCache, err := newRowFetcherCache(
 		ctx,
 		cfg.Codec,
 		cfg.LeaseManager.(*lease.Manager),
 		cfg.CollectionFactory,
 		cfg.DB,
 		details,
 	)
+	if err != nil {
+		logcrash.ReportOrPanic(ctx, &cfg.Settings.SV,
+			`error creating rowfetcher cache for details`,
+			details)
+	}
```

Comment:
Looking at release branch, I see exactly 2 call sites to newKVEventTORowConsumer;
Let's just return an error?